.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "http://json.schemastore.org/prettierrc",
+  "printWidth": 100,
```

Comment:
prettier recommends against setting this:
https://prettier.io/docs/en/options.html

is there a good reason for setting it higher?

Reply:
you mean higher printWidth causes lines to break up less, right? not sure why a higher printWidth would reduce the number of long, unbroken lines

Reply:
yes but then there will be extra cases that raises eyebrows, or so was our experience in content which led to 100 as a seemingly reasonable alternative